README.md

```diff
@@ -43,19 +43,21 @@ source devel/setup.bash
 
 ### Parameters:
  [config_file:"<<b>path></b>"]
-* Crosby with Pulsatrix: "```config/config_walden_3500_nxp.json```"
-* Crosby: "```config/config_crosby_nxp.json```"
-* Walden: "```config/config_walden_nxp.json```"
+* Crosby with Pulsatrix: "```tof_config/config_crosby_adsd3500_new_modes.json```"
+* Tembin with Pulsatrix: "```tof_config/config_crosby_adsd3500_adsd3030_new_modes.json```"
```

Comment:
the 'crosby' name doesn't make sense. This .json is for tembin, right?

Reply:
removed